Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2012-2014 Gabriel Gonzalez
```

Comment:
Here you go:

https://github.com/Gabriel439/Haskell-Pipes-Library/commit/9e2467f95db400ff7701439e3e2b54e06e4177b3